### PR TITLE
Browser: Add CSS based Adblock plugin

### PR DESCRIPTION
**Pull request by HollowMan6**
Add the CSS-based Adblock functionality for browser.

An example of Adblock CSS is provide at `core/adblock.css` from http://www.floppymoose.com/ , users can customize their own rules.

Use `C-d` to enable or disable adblock.

Here is a demo which shows the effect of block Google ADs when enable and disable adblock at my [website](https://hollowman6.github.io/fund.html) under the provided CSS file:
image
image

### CODE REVIEW

File: core/browser.py [hunk 1]
```diff
@@ -51,6 +51,7 @@ def __init__(self, buffer_id, config_dir):
         self.installEventFilter(self)
         self.buffer_id = buffer_id
         self.config_dir = config_dir
+        self.enable_adblock = True
```
Comment (MatthewZMD) [inline core/browser.py:54]: 放到eaf-vaf-list里面，做成全局的一个参数

Comment (HollowMan6) [reply core/browser.py]: OK, 更新完成！

File: eaf.el [hunk 2]
```diff
@@ -242,6 +242,7 @@ It must defined at `eaf-browser-search-engines'."
 (defcustom eaf-var-list
   '((eaf-camera-save-path . "~/Downloads")
     (eaf-browser-enable-plugin . "true")
+    (eaf-browser-enable-adblock . "true")
```
Comment (MatthewZMD) [inline eaf.el:245]: 默认值设为false比较好

**Comment by MatthewZMD** [comment]
现在有个问题，`x`尝试关闭页面会在`*eaf*` buffer弹出以下错误
```
Traceback (most recent call last):
  File "/home/mt/.emacs.d/site-elisp/emacs-application-framework/app/browser/buffer.py", line 66, in set_adblocker
    self.buffer_widget.load_adblocker()
  File "/home/mt/.emacs.d/site-elisp/emacs-application-framework/core/browser.py", line 107, in load_adblocker
    self.load_css(os.path.join(os.path.dirname(__file__), "adblocker.css"),'adblocker')
  File "/home/mt/.emacs.d/site-elisp/emacs-application-framework/core/browser.py", line 98, in load_css
    self.web_page.runJavaScript(SCRIPT, QWebEngineScript.ApplicationWorld)
RuntimeError: wrapped C/C++ object of type BrowserPage has been deleted

```

**Pull request merged**